smbus2/smbus2.py

```diff
@@ -316,6 +317,18 @@ def close(self):
             os.close(self.fd)
             self.fd = None
 
+    def enable_pec(self, enable=True):
```

Comment:
Thanks, @rgusmero. I have a couple of comments I hope you don't mind to take care of. The first one is that I would like the interface of `smbus2` to match that of smbus-cffi to keep the drop-in replacement alternative intact. Thus, I would like you to change the method name and header to
```Python
pec(self, enable):
```
and let any non-zero value be interpreted as `True`.